### PR TITLE
readme: add yarn, pnpm, bun and deno

### PR DESCRIPTION
**Pull request by 6km**
I have updated the README to add instructions for yarn, pnpm, bun and deno.
also changed npm command to match the docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,21 @@
 
 You can start to build a Hono application with `create hono` command:
 
-```
+```bash
+# npm
 npm create hono@latest
+
+# yarn
+yarn create hono my-app
```
Comment (yusukebe) [inline README.md:10]: What about removing `my-app`?

```bash
yarn create hono
```

File: README.md [hunk 2]
```diff
@@ -2,8 +2,21 @@
...
+
+# pnpm
+pnpm create hono my-app
```
Comment (yusukebe) [inline README.md:13]: Adding a `latest` is better:

```bash
pnpm create hono@latest
```

File: README.md [hunk 3]
```diff
@@ -2,8 +2,21 @@
...
+
+# bun
+bunx create-hono my-app
```
Comment (yusukebe) [inline README.md:16]: I think the following is better.

```bash
bun create hono@latest
```

File: README.md [hunk 4]
```diff
@@ -2,8 +2,21 @@
...
+
+# deno
+deno run -A npm:create-hono my-app
```
Comment (yusukebe) [inline README.md:19]: We can add a `latest` also for Deno:

```bash
deno run -A npm:create-hono@latest
```

**Comment by yusukebe** [comment]
Hi @6km 

Thank you for the PR!

> also changed npm command to match the docs.

I've left some comments. I think the commands on my comments are the best, though they are different from the docs (we should update that).

**Comment by 6km** [comment]
Thanks for your review, @yusukebe!
I think the commands are better now with your comments!

**Review by yusukebe — APPROVED** [review]
LGTM!

**Comment by yusukebe** [comment]
@6km 

Thanks for changing them! Let's go with it.

**Pull request merged**